core/emulate.c

```diff
@@ -854,10 +905,49 @@ em_status_t EMCALL em_decode_insn(struct em_context_t *ctxt, const uint8_t *insn
             b = insn_fetch_u8(ctxt);
             *opcode = opcode_table_0F3A[b];
             break;
-
         default:
             *opcode = opcode_table_0F[b];
         }
+        break;
+
+    case 0xC4:
+    case 0xC5:
+        /* Intel SDM Vol. 2A: 2.3.5 The VEX Prefix */
+        ctxt->vex.prefix = b;
+        if (b == 0xC4) {
+            ctxt->vex.value = insn_fetch_u16(ctxt);
+        } else {
+            ctxt->vex.value = insn_fetch_u8(ctxt);
```

Comment:
[Unsure] Should the right-hand side be `insn_fetch_u8(ctxt) << 8` instead?

Reply:
True, thank you. My brain was accidentally in big-endian mode. :-)

Reply:
:-) I was wondering why the unit tests didn't catch this. Now I see--both `ANDN` and `BEXTR` require a 3-byte VEX prefix...